crates/router/src/connector/noon/transformers.rs

```diff
@@ -334,11 +336,11 @@ impl From<NoonPaymentStatus> for enums::AttemptStatus {
             NoonPaymentStatus::Authorized => Self::Authorized,
             NoonPaymentStatus::Captured | NoonPaymentStatus::PartiallyCaptured => Self::Charged,
             NoonPaymentStatus::Reversed => Self::Voided,
-            NoonPaymentStatus::Cancelled => Self::AuthenticationFailed,
+            NoonPaymentStatus::Cancelled | NoonPaymentStatus::Expired => Self::AuthenticationFailed,
```

Comment:
can we use `AuthorizationFailed`?

Reply:
No since it goes to `requires_customer_action` flow where the user fails to authenticate within a given time before the link gets expired, `authenticationFailed` is a more appropriate error